### PR TITLE
Add missing and rearrange options in .conf

### PR DESCRIPTION
**Pull request by Faalagorn**
Added missing options and rearranged the way they appear under legacy layout plus minor fixes:
* Added some default values after `=`
* Commented out options with default values
* Unified style

### CODE REVIEW

**Comment by Faalagorn** [comment]
This is mostly so that after enabling `legacy_layout=false` things should work as usual.

File: bin/MangoHud.conf [hunk 1]
```diff
@@ -143,32 +166,28 @@ background_alpha=0.5
...
 
-### (Re)bind given framebuffer before MangoHud gets drawn. Helps with Crusader Kings III.
-# gl_bind_framebuffer = 0
+### (Re)bind given framebuffer before MangoHud gets drawn. Helps with Crusader Kings III
+# gl_bind_framebuffer=0
 
-### Don't swap origin if using GL_UPPER_LEFT. Helps with Ryujinx.
-# gl_dont_flip = 1
+### Don't swap origin if using GL_UPPER_LEFT. Helps with Ryujinx
+# gl_dont_flip=1
```
Comment (jackun) [inline bin/MangoHud.conf:188]: Spaces don't cost extra though

Comment (Faalagorn) [reply bin/MangoHud.conf:188]: I don't mind having extra space as long as the style is uniform, I think there was something that made me decide on having them removed, but I don't know what exactly. Do you want me to change them all to having spaces?

Comment (jackun) [reply bin/MangoHud.conf:188]: Eh, let it be

**Comment by jackun** [comment]
In 87e25a3038f93be1ef7dc47a39ef056a769d7bd1

**Pull request closed without merging**